### DISCUSSION TITLE
中央通識修課經驗-認識地球

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1779] [General]
最近看到有人在po語言課的修課經驗，就想說來分享一下通識課的經驗

認識地球-陳台琦  , 王乾盈  , 朱延祥  , 郭陳澔
這堂課上課都不用點名，作業也才少少的只有2次，而且負擔都沒有很大，但有期中期末考，不過只要會高中地科再在考前複習一下ppt內容就可以考很高分了，如果你拿的到考古題的話這堂課的考試大部分題目也都跟考古差不多，最終我學期成績拿到97分，是又涼又甜的課。

**Comment by Ryanlolry** [comment]
我也修過這堂課，老師上課是蠻認真的，且考試雖然有些難度，我也都不太會寫，可是在沒考古的情況下，也拿的蠻高分的，推薦想拿高分的人跟想認真學地球科學的人來修這堂課。
